### PR TITLE
Squelch robocopy output

### PR DESCRIPTION
**Pull request by melezov**
Lowers the noise of #173

### CODE REVIEW

File: src/universal/bin/sbt.bat [hunk 1]
```diff
@@ -159,7 +159,7 @@ if /I "%JAVA_VERSION%" GEQ "1.8" (
     if not exist %PRELOAD_SBT_JAR% (
       if exist "%SBT_HOME%\..\lib\local-preloaded\" (
         echo 'about to robocopy'
-        robocopy "%SBT_HOME%\..\lib\local-preloaded" "%UserProfile%\.sbt\preloaded" /E
+        robocopy /NS /NC /NFL /NDL /NP /NJH /NJS "%SBT_HOME%\..\lib\local-preloaded" "%UserProfile%\.sbt\preloaded" /E
```
Comment (dwijnand) [inline src/universal/bin/sbt.bat]: From https://technet.microsoft.com/en-us/library/cc733145(v=ws.11).aspx:

```
/ns     Specifies that file sizes are not to be logged.
/nc     Specifies that file classes are not to be logged.
/nfl    Specifies that file names are not to be logged.
/ndl    Specifies that directory names are not to be logged.
/np     Specifies that the progress of the copying operation
          (the number of files or directories copied so far) will not be displayed.
/njh    Specifies that there is no job header.
/njs    Specifies that there is no job summary.
```

However at the top it says the syntax is:

```
robocopy <Source> <Destination> [<File>[ ...]] [<Options>]
```

Are you sure you can put the options at the beginning?

Comment (melezov) [inline src/universal/bin/sbt.bat]: Hi, pretty sure but it's no biggie to move them according to spec. Sometimes these things are version dependent so it' s better to do it right.

**Comment by eed3si9n** [comment]
I didn't intend robocopy to run on every invocation of `sbt.bat`, so if that's happening I don't think we should sweep it under the rug by quieting robocopy.

**Comment by eed3si9n** [comment]
Closing this in favor of https://github.com/sbt/sbt-launcher-package/pull/175

**Pull request closed without merging**